src/Synapse/Synapse/Models/DataPlaneModels/Artifact/LinkConnection/PSLinkConnectionTargetDatabaseTypeProperties.cs

```diff
@@ -22,10 +22,13 @@ public PSLinkConnectionTargetDatabaseTypeProperties(LinkConnectionTargetDatabase
         {
             this.CrossTableTransaction = properties?.CrossTableTransaction;
             this.DropExistingTargetTableOnStart = properties?.DropExistingTargetTableOnStart;
+            this.ActionOnExistingTargetTable = properties?.ActionOnExistingTargetTable;
         }
 
         public bool? CrossTableTransaction { get; set; }
 
         public bool? DropExistingTargetTableOnStart { get; set; }
+
+        public ActionOnExistingTargetTable? ActionOnExistingTargetTable { get; set; }
```

Comment:
does `ActionOnExistingTargetTable` also SDK model? if yes, you also need to add wrapper class.

Reply:
ActionOnExistingTargetTable on SDK is public readonly partial struct ActionOnExistingTargetTable : IEquatable<ActionOnExistingTargetTable>. It's not a class.